.github/workflows/galaxy-publish-collection.yml

```diff
@@ -0,0 +1,27 @@
+name: galaxy-publish-collection
+on:
+  push:
+    paths:
+    - 'os_migrate/galaxy.yml'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt update -y
+        sudo apt install ansible jq -y
+        python -m pip install --upgrade pip
+        pip install tox shyaml
```

Comment:
Similarly as jq installation above, these should also go to venv-requirements.txt.
https://github.com/os-migrate/os-migrate/blob/1b5c8c3baddda8def35035706cab97cce2c0ad84/toolbox/build/venv-requirements.txt#L4

Reply:
done